engarde/checks.py

```diff
@@ -246,7 +247,28 @@ def one_to_many(df, unitcol, manycol):
     return df
 
 
-__all__ = ['is_monotonic', 'is_shape', 'none_missing', 'unique_index', 'within_n_std',
-           'within_range', 'within_set', 'has_dtypes',
-           'verify', 'verify_all', 'verify_any']
+def is_same_as(df, df_to_compare):
```

Comment:
Can you accept `**kwargs` here and pass them through to the `tm.assert_frame_equal`? That can be useful for relaxing the equality requirements.